src/main/java/by/bookify/genre/exception/GenreExceptionHandler.java

```diff
@@ -0,0 +1,16 @@
+package by.bookify.genre.exception;
+
+import by.bookify.general.model.ExceptionResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class GenreExceptionHandler {
+
+    @ExceptionHandler(GenreNotFound.class)
+    public ResponseEntity<ExceptionResponse> handleGenreNotFound(GenreNotFound e) {
+        return new ResponseEntity<>(new ExceptionResponse(e.getMessage()), HttpStatus.NOT_FOUND);
```

Comment:
Code style issue. In controllers you are using static methods of ResponseEntity, but in exception handlers you instantiate through constructor

Reply:
The others adviceControllers use different codes where there are no statics methods for some codes. What to do?

Reply:
ResponseEntity.status()

Reply:
Done